src/diffusion_for_multi_scale_molecular_dynamics/score/gaussian_score.py

```diff
@@ -0,0 +1,47 @@
+"""Gaussian Score.
+
+This module implements the Gaussian score, which corresponds to the "perturbation kernel" for diffusion on the lattice
+parameters.
+
+The Gaussian takes the form
+
+    K(l, l0) ~ exp[- |l - l0|^2 / 2 sigma^2]
+
+The score is defined as S = nabla_l ln K(l, l0).
+"""
+
+import torch
+
+
+def get_lattice_sigma_normalized_score(
+    delta_l: torch.Tensor, sigma_n: torch.Tensor, alpha_bar: torch.Tensor
+) -> torch.Tensor:
+    r"""Get the sigma normalized score for lattice parameters.
+
+    We compute this from a normal (non-wrapped) gaussian kernel.  Because we are using a variance-preserving approach
+    for the lattice parameters, the width of the gaussian kernel is rescaled by the noise parameter :math:`alpha_bar`.
+
+    .. math::
+
+        \sigma_\textrm{eff}^2 = (1 - \bar{\alpha}) \sigma^2(n)
+
+    with :math:`\sigma(n) = \frac{\sigma}{\sqrt[1/d]{n}}` with :math:`d` the number of spatial dimensions.
+
+    Args:
+        delta_l : the lattice parameters at which the wrapped Gaussian is evaluated.
```

Comment:
We are not using wrapped Gaussians here. Copy-paste malfunction?

Reply:
correct - this input variable was replaced by noisy_l and real_l